commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -128,44 +132,82 @@ protected void validateArgs() {
     validatePositiveValue(
         slashingProtectionParameters.getPruningSlotsPerEpoch(), "Pruning slots per epoch");
 
+    validateAzureParameters();
+    validateAwsParameters();
+  }
+
+  private void validateAzureParameters() {
```

Comment:
Same comment as above, changes to the subcommand aren't needed for single AWS key loading. We only have command line options for bulk loading mode.